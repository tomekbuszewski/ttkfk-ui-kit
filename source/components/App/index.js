import React from 'react';
import styles from './style.scss';

const App = () => <div className={styles.main}>Hello</div>;

export default App;