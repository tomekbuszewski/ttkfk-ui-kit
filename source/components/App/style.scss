.main {
  background: darken(#fff, 30%);
}