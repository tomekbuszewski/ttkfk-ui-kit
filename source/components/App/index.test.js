import React from 'react';
import renderer from 'react-test-renderer';

import App from './index';

describe ('App test', () => {
  it ('should match snapshot', () => {
    const tree = renderer.create(<App />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
