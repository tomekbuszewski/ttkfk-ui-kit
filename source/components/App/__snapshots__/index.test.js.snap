// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test should match snapshot 1`] = `
<div
  className="main"
>
  Hello
</div>
`;
